test/server/component.js

```diff
@@ -68,8 +68,9 @@ describe(`Server-side component renderer`, function () {
   });
 
   it(`renders nested components`, async function () {
+    document.body = document.createElement(`body`);
```

Comment:
Need to do this a lot eh? Maybe it should happen in a `beforeEach`, let tests count on `document.body` being available (and fresh)